.readthedocs.yaml

```diff
@@ -11,6 +11,12 @@ build:
   tools:
     python: "3.11"
   jobs:
+    pre_install:
+      # ECMWF credentials
+      - bash scripts/set-readthedocs-ecmwf-credentials.sh
+      - echo $ECMWF_DATASTORES_URL
+      - echo $HOME
+      - cat "$HOME/.ecmwfdatastoresrc"
```

Comment:
```suggestion
```

Remove before merging (otherwise your credentials leak to anyone who sees the logs)